api/src/main/kotlin/edu/wgu/osmt/jobcode/JobCodeRepository.kt

```diff
@@ -52,6 +54,20 @@ class JobCodeRepositoryImpl: JobCodeRepository {
             .firstOrNull()?.let { dao.wrapRow(it) }
     }
 
+    override fun createFromApi(jobCodes: List<JobCodeUpdate>): List<JobCodeDao> {
+        return jobCodes.map { jobCodeUpdate ->
+            dao.new {
+                this.code = jobCodeUpdate.code
+                this.framework = jobCodeUpdate.framework
+                this.name = jobCodeUpdate.targetNodeName
+                this.creationDate = LocalDateTime.now(ZoneOffset.UTC)
+                // this.description = jobCodeUpdate.description
```

Comment:
Is this line being saved for a follow-up admin screen ticket, or should it be removed?

Reply:
I removed the line since also was removed from `JobCodeUpdate`